### PR TITLE
Document what it looks like to use a `conda-lock` environment file

### PR DESCRIPTION
**Pull request by mfisher87**
On my first attempt to use this action with a conda-lock file, I encountered a fairly [unintuitive error message](https://github.com/nsidc/qgreenland/actions/runs/5216719569/jobs/9415735414). I discovered I just needed to specify an environment name, and I thought this doc change might help someone else out there :)

### CODE REVIEW

**Comment by pavelzw** [comment]
Thanks for the addition!

What setup-micromamba does internally is search for a `name: ...` in the `environment.yml` file. If it doesn't find anything and `environment-name` is not set, an error is raised. In a lock file, though, the action can find a `name: ...` and thinks incorrectly that the environment name is set.

File: README.md [hunk 1]
```diff
@@ -49,6 +49,18 @@ This will automatically activate the environment when the shell is started.
     environment-file: environment.yml
 ```
 
+#### Create environment from `conda-lock` file
+
+Mamba supports the `conda-lock` [unified lockfile format](https://conda.github.io/conda-lock/output/#unified-lockfile), but this format does not specify an environment name.
```
Comment (pavelzw) [inline README.md:54]:
```suggestion
Micromamba supports the `conda-lock` [unified lockfile format](https://conda.github.io/conda-lock/output/#unified-lockfile), but this format does not specify an environment name.
```

**Review by pavelzw — APPROVED** [review]

**Comment by pavelzw** [comment]
Thanks @mfisher87!

**Comment by jonashaag** [comment]
@pavelzw I think we should add a `^` to the regex here https://github.com/mamba-org/setup-micromamba/blob/cc9a6fb8a9ffb265351b3a292bf890b96c934e9b/src/util.ts#L53

**Comment by pavelzw** [comment]
That's a good idea!

**Pull request merged**